danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/Utils.kt

```diff
@@ -17,15 +18,10 @@ class Utils {
      * Gives you the ability to cheaply run a command and read the output without having to mess around
      *
      * @param command The first part of the command
-     * @param arguments An optional array of arguments to pass in extra
+     * @param arguments An optional list of arguments to pass in extra
      * @return the stdout from the command
      */
-    fun exec(command: String, arguments: Array<String> = arrayOf()): String {
-        val commandWithArgs = command + if (arguments.isNotEmpty())  " " + arguments.joinToString(" ") else ""
-
-        val process = Runtime.getRuntime().exec(arrayOf("/bin/bash", "-c", commandWithArgs))
-        process.waitFor()
-
-        return process.inputStream.bufferedReader().readText()
+    fun exec(command: String, arguments: List<String> = emptyList()): String {
+       return ShellExecutorFactory.get().execute(command, arguments)
```

Comment:
i don't really like the idea of proxy here, we can directly use utils

Reply:
well that was what we discussed with @f-meloni in my previous PR 🤔